numpy/distutils/fcompiler/intel.py

```diff
@@ -27,6 +27,17 @@ def runtime_library_dir_option(self, dir):
         return '-Wl,-rpath=%s' % dir
 
 
+    def get_version(self):
+        if platform.system() == 'Windows':
+            version_cmd = "icl -dummy"
+            status, output = exec_command(version_cmd, use_tee=0)
```

Comment:
I believe `exec_command` is now deprecated, and this should either use self.spawn or subprocess.

Reply:
@oleksandr-pavlyk exec_command was deprecated in 1.17.

Reply:
@oleksandr-pavlyk Ping.

Reply:
I pushed the change to use `subprocess`.